web/static/js/components/idea_edit_form.jsx

```diff
@@ -25,14 +32,29 @@ class IdeaEditForm extends Component {
   onSubmit(event) {
     event.preventDefault()
     const { idea, retroChannel } = this.props
-    const { ideaBody } = this.state
-    retroChannel.push("idea_edited", { id: idea.id, body: ideaBody })
+    const { ideaBody, ideaCategory } = this.state
+    retroChannel.push("idea_edited", {
+      id: idea.id,
+      body: ideaBody,
+      category: ideaCategory,
+    })
   }
 
   render() {
+    const { categories, stage } = this.props
     return (
       <form onSubmit={this.onSubmit} className="ui form">
         <p className="ui center aligned sub header">Editing</p>
+        {stage !== "action-items" && <select
```

Comment:
let's get a unit test around this conditional render

Reply:
done